docs/sharing.md

```diff
@@ -42,6 +46,8 @@ Do not enable the `Make dataset publicly accessible` checkbox or otherwise, your
 Public access rights are not required for private sharing.
 {% endhint %}
 
+{% embed url="https://www.youtube.com/watch?v=hcm8Jx22DG8 "%}
```

Comment:
This would fit better into the "Annotation Sharing" section further down as the video shows the sharing of a large annotation for a publication :)